pkg/tsdbctl/delete.go

```diff
@@ -55,8 +55,8 @@ func newDeleteCommandeer(rootCommandeer *RootCommandeer) *delCommandeer {
 
 	cmd.Flags().BoolVarP(&commandeer.delConfig, "del-config", "d", false, "Delete the TSDB config as well")
 	cmd.Flags().BoolVarP(&commandeer.force, "force", "f", false, "Delete all elements even if some steps fail")
-	cmd.Flags().StringVarP(&commandeer.toTime, "end", "e", "", "to time")
-	cmd.Flags().StringVarP(&commandeer.fromTime, "begin", "b", "", "from time")
+	cmd.Flags().StringVarP(&commandeer.toTime, "end", "e", "0", "to time")
```

Comment:
end time could be set to `now` by default if we want to delete entire table if no boundaries set